src/code.rs

```diff
@@ -153,32 +168,39 @@ impl<'a> Struct<'a> {
 
     /// Assemble all derives for the struct
     fn attr_derive(&self) -> String {
-        format!("#[derive(Debug, {derive_serde}Clone, Queryable, Insertable{derive_aschangeset}{derive_identifiable}{derive_associations}{derive_selectable}{derive_default})]",
-                derive_selectable = match self.ty {
-                    StructType::Read => { ", Selectable" }
-                    _ => { "" }
-                },
-                derive_associations = match self.ty {
-                    StructType::Read => {
-                        if !self.table.foreign_keys.is_empty() { ", Associations" } else { "" }
-                    }
-                    _ => { "" }
-                },
-                derive_identifiable = match self.ty {
-                    StructType::Read => {
-                        if !self.table.foreign_keys.is_empty() { ", Identifiable" } else { "" }
-                    }
-                    _ => { "" }
-                },
-                derive_aschangeset = if self.fields().iter().all(|f| self.table.primary_key_column_names().contains(&f.name)) {""} else { ", AsChangeset" },
-                derive_default = match self.ty {
-                    StructType::Update => { ", Default" }
-                    _ => { "" }
-                },
-                derive_serde = if self.config.table(&self.table.name.to_string()).get_serde() {
-                    "Serialize, Deserialize, "
-                } else { "" }
-        )
+        let mut derives_vec = Vec::with_capacity(10);
```

Comment:
question: should we make the capacity a const, in the derives module as well? That way the code is closer and if someone adds another derive, they can update it there (it's easier to notice)?

Reply:
not a big deal though, feel free to ignore this!

Reply:
i dont really think it is necessary to make it a const for that, because it is basically just a suggestion and only a really minor improvement that might as well be not there, i just added it because i had some extra information and so could make a good guess.